src/ontology/cl-edit.owl

```diff
@@ -27096,6 +27110,7 @@ SubClassOf(obo:CL_1001127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000477
 
 # Class: obo:CL_1001131 (vasa recta ascending limb cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20392798"^^xsd:string) obo:IAO_0000115 obo:CL_1001131 "A cell that is part of the vasa recta ascending limb. It is known in some mammalian species that this cell may express PV-1, a component of the fenestral diaphragm.")
```

Comment:
The linked paper seems to focus on the outer zone of the inner medulla, so I'm not sure if the second sentence is always applicable to this cell type, but this may be OK since "may express" is written.